engine-rest/engine-rest-openapi/src/main/templates/paths/task/get.ftl

```diff
@@ -38,7 +38,7 @@
         array = true
```

Comment:
🔧 I think in the previous role we need to change the dto to `TaskWithAttachmentAndCommentDto`.
Same is valid for the other endpoints in this PR.